docs/production/system-configuration.md

```diff
@@ -257,6 +257,18 @@ may wish to increase this if you are taking backups on a replica, so can afford
 to affect other disk I/O, and have an SSD which is good at parallel random
 reads.
 
+#### `backups_directory`
+
+If S3 secrets are not configured, perform daily database backups to this path on
+disk instead. It should be owned by the `postgres` user.
+
+Note that because this is likely on the same disk as the database itself, this
+_will not serve any disaster recovery purpose_, since these backups will likely
```

Comment:
This may be a bit strong, since it could be a network-mounted volume.

Reply:
Sure.  Repushed!